src/tables/interface.jl

```diff
@@ -0,0 +1,188 @@
+struct SeriesTable <: Tables.AbstractColumns
+    attrs::Dict{Symbol,Int}
+    types::Vector{Type}
```

Comment:
Why?
Agents are always floats. stage, scenario and block are always Ints

Reply:
A `SeriesTable` doesn't represent a graf file. It is intended for represented mixed-type series suchs as:
```julia
series_data = Dict(
        "PSRThermalPlant" => [
            "Data"     => Dict{String,Vector}(
            "GerMin"   => Float64[0.0, 1.0],
            "GerMax"   => Float64[888.0, 777.0],
            "O&MCost"  => Float64[0.0, 1.0],
            "IH"       => Float64[0.0, 0.0],
            "ICP"      => Float64[0.0, 0.0],
            "Data"     => Dates.Date.(["1900-01-01", "1900-01-02"]),
            "CoefE"    => Float64[1.0, 2.0],
            "CTransp"  => Float64[0.0, 1.0],
            "PotInst"  => Float64[888.0, 777.0],
            "Existing" => Int32[0, 0],
            "sfal"     => Int32[0, 1],
            "NGas"     => Int32[0, 0],
            "NAdF"     => Int32[0, 0],
            "Unidades" => Int32[1, 1],
            "StartUp"  => Float64[0.0, 2.0],
            ),
        ],
```